src/state/sync/UserAssetsSync.tsx

```diff
@@ -9,28 +8,27 @@ import { ChainId } from '@/__swaps__/types/chains';
 import { getIsHardhatConnected } from '@/handlers/web3';
 import { useUserAssets } from '@/__swaps__/screens/Swap/resources/assets';
 
-export const UserAssetsSync = function UserAssetsSync() {
-  const { accountAddress: currentAddress, nativeCurrency: currentCurrency } = useAccountSettings();
+export const UserAssetsSync = memo(function UserAssetsSync() {
```

Comment:
for whatever reason I noticed the memo here messes with e2e switching to hardhat assets

Reply:
oh ok, i saw it got removed in brodys pr so i asked him about that but hadn't heard back yet